### DISCUSSION TITLE
「未解決」「解決済み」に限らず特定期間のインシデントの一覧を出したい

### DISCUSSION THREAD

**Discussion by kenta-hi** [topotal/waroom-roadmap#64] [Feature Request]
## Why

現在、期間を絞ってインシデントを表示する際に「未解決」「解決済み」に分かれてしまっている
「未解決」「解決済み」に限らず特定期間のインシデントの一覧を出したい
インシデントを振り返る際に一覧として出力されて、ステータスはラベル的に見えた方がスムーズに振り返りが実施できる

## What

期間を絞ってインシデントを表示する際にステータスによらず一覧を表示する

**Comment by sawa-zen** [comment]
タイトルを`「未解決」「解決済み」に限らず特定期間のインシデントの一覧を出したい` に変更させていただきました。

**Comment by gr1m0h** [comment]
元の要望を出した者ですが、インシデント一覧ページで「未解決」「解決済」に加えて「すべて」というタブの追加がリリースされているので、本DiscussionはCloseで良いかと思いました
@kenta-hi @sawa-zen

**Reply by sawa-zen** [reply]
確認ありがとうございます！こちらクローズしますね 👍🏻
